app/plugin_api.go

```diff
@@ -315,14 +315,6 @@ func (api *PluginAPI) UpdateUserStatus(userID, status string) (*model.Status, *m
 	return api.app.GetStatus(userID)
 }
 
-func (api *PluginAPI) SetUserStatusTimedDND(userID string, endTime int64) (*model.Status, *model.AppError) {
```

Comment:
Are there any docs attached to this that we might need to revert? I'd assume this hasn't been around long enough to worry that we're breaking anyone's plugin though.